python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -327,6 +348,14 @@ def is_multipartitioned(self) -> bool:
             external_multipartitions_def, ExternalMultiPartitionsDefinitionData
         )
 
+    def convert_stale_status(self, status: StaleStatus) -> GrapheneAssetStaleStatus:
+        if status == StaleStatus.FRESH:
+            return GrapheneAssetStaleStatus.FRESH  # type: ignore  # (bad stubs)
+        elif status == StaleStatus.STALE:
+            return GrapheneAssetStaleStatus.STALE  # type: ignore  # (bad stubs)
+        else:
+            return GrapheneAssetStaleStatus.UNKNOWN  # type: ignore  # (bad stubs)
+
```

Comment:
I'd defer to @sryza but I think there's some sort of magic incantation you can do to have this conversion done automatically, maybe renaming `GrapheneAssetStaleStatus` to `GrapheneStaleStatus` so it matches the python enum name would do it?

Reply:
I was wondering about this...

Reply:
I think there may be, but I don't know what it is. @prha or @alangenfeld or @clairelin135 might?

Reply:
can try `from_enum` https://docs.graphene-python.org/en/latest/types/enums/